Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -662,6 +729,23 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
       return color;
     }
 
+    /// <summary>
+    /// Lerps this color towards the argument color in HSV space and returns the lerped
+    /// color.
+    /// </summary>
+    public static Color LerpHSV(this Color color, Color towardsColor, float t) {
+      float h0, s0, v0;
+      Color.RGBToHSV(color, out h0, out s0, out v0);
+
+      float h1, s1, v1;
+      Color.RGBToHSV(towardsColor, out h1, out s1, out v1);
+
+      float hL = Mathf.Lerp(h0, h1, t);
```

Comment:
Would probably be good to do something special here for hue, since it is cyclic.  If we have a hue of 0.05 and lerp to a hue of 0.95 it will lerp across the _entire_ rainbow instead of just through the small red portion that separates the two hues.